### ISSUE TITLE
Use Selenium method to clear input field

### ISSUE THREAD

**Issue by mkusz** [pawel-marek/master#1]
There is for sure method for cleaning input filed. Do not use key send combination for CTRL+A.
https://github.com/pawel-marek/master/blob/20ab0e6813251a8081038b6a2df7df2622b4f1a7/Testowanie_p/POM_Sites/FindOwners.cs#L29

**Comment by pawel-marek** [comment]
I was trying to use method 'Driver.Clear()' - but in my case when in the input field was some text this method simply didn't work correctly, the text wasn't removed. After using 'CTRL+A' gave me a expected results.

**Comment by mkusz** [comment]
Here you have an example how to do it: https://stackoverflow.com/questions/15167429/what-does-webelement-clear-do-to-textboxes

**Comment by mkusz** [comment]
Yes. Read last error. It says that element is not editable. Try to add something like wait to be clicable before trying to clear value of the field or even better try to use moveToElement from Actions. It will do 2 things: scroll to an element even if it's outside browser viewport and get focus on that element. Try to implement it as an function decorator so it will be reusable function.
https://stackoverflow.com/questions/11337353/correct-way-to-focus-an-element-in-selenium-webdriver-using-java

**Comment by pawel-marek** [comment]
I'm not sure if I'm doing exactly what you expected, but some time ago I was adding new static method (you can find it in DriverExtensions class; method name: SendKeys) for sending the text to input fields and after reading your last comments I realised, that I can use it as well. It works fine from my perspective, but i'm wondering what do you think about this method from your perspective?

**Comment by mkusz** [comment]
My bad. C# do not have function decorators like Python (didn't know that). https://stackoverflow.com/questions/15323009/implement-method-decorators-in-c-sharp
Probably you can achieve it differently.
